README.md

```diff
@@ -72,8 +86,11 @@ class MyDaemonizedCommand extends DaemonCommand
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $output->writeln("Iteration");
-
-        usleep(100000);
+        
+        // This method helps to give back the CPU to the react-loop (here for 1000 $µseconds).
+        // So you can wait between two iterations if your workers has nothing to do.
+        
+        $this->setNextIterationSleepingTime(1000000); // Every seconds
```

Comment:
I'm not so sure about that but I think the word after `every` should not be pluralized... 

I think 👉  @pabedu 👈  taught me that (you can hit him if it's wrong)